docs/coding-style.md

```diff
@@ -27,6 +27,8 @@ Static fields and properties should be placed before instance ones.
 
 Static methods are preferred to be after instance methods.
 
+Individual members must be separated by empty line, except sequence of constants, fields and abstract members. These should not be separated by empty lines.
```

Comment:
@costin-zaharia-sonarsource This is new. I realized we didn't have it documented.